src/util/toxic.hpp

```diff
@@ -0,0 +1,367 @@
+#pragma once
+#include "pointers.hpp"
+#include "core/scr_globals.hpp"
+#include "core/enums.hpp"
+#include "gta/net_object_mgr.hpp"
+#include "gta/PickupRewards.h"
+#include "util/session.hpp"
+#include "util/scripts.hpp"
+#include "services/gta_data/gta_data_service.hpp"
+#include "util/system.hpp"
+
+namespace big::toxic
+{
+	inline void blame_explode_coord(player_ptr to_blame, Vector3 pos, eExplosionTag explosion_type, float damage, bool is_audible, bool is_invisible, float camera_shake)
+	{
+		system::patch_blame(true);
```

Comment:
Patch Blame could be swapped to the new memory::byte_patch.

Same for despawn bypass.